source/mainnet/net/guides/developer-page.rst

```diff
@@ -32,35 +21,48 @@ Official resources
+    - :ref:`Concordium gRPC V2 API<grpc2-documentation>`
+    - `Concordium Rust SDK <https://github.com/Concordium/concordium-rust-sdk>`_
+    - `Concordium Javascript (Node / Web) SDK <https://github.com/Concordium/concordium-node-sdk-js>`_
+    - `Concordium Java SDK <https://github.com/Concordium/concordium-java-sdk>`_
+    - `Concordium .NET (C#) SDK <https://github.com/Concordium/concordium-net-sdk>`_
+    - `Concordium Rosetta SDK <https://github.com/Concordium/concordium-rosetta>`_
+    - `Concordium go SDK <https://github.com/Concordium/concordium-go-sdk>`_
 
-You can watch a video about how to use the VSCode extension.
+    The following NPM libraries are useful for building web-based dApps:
 
-.. image:: https://img.youtube.com/vi/9qjcsGDeveg/maxresdefault.jpg
-    :alt: video about smart contract lifecycle
-    :target: https://www.youtube.com/watch?v=9qjcsGDeveg
+    - `@concordium/react-components <https://www.npmjs.com/package/@concordium/react-components>`_ (for React projects).
+    - `@concordium/wallet-connectors <https://www.npmjs.com/package/@concordium/wallet-connectors>`_ (for connecting to wallets in non-React projects).
+    - `@concordium/web-sdk <https://www.npmjs.com/package/@concordium/web-sdk>`_ (for interacting with a chain).
+    - `@concordium/ccd-js-gen <https://www.npmjs.com/package/@concordium/ccd-js-gen>`_ (for generating smart contract clients in JavaScript).
 
-- An external `transaction logger <https://github.com/Concordium/concordium-transaction-logger>`_ service
+.. dropdown:: Smart contracts
 
-Concordium Proof Explorer
--------------------------
+    See :ref:`Smart contracts <introduction>` for general information about smart contracts, including a list of the available :ref:`smart contract development tools<sc-dev-tools>` to make creation, testing, and deployment easier.
+
+    - `Smart contract libraries <https://crates.io/crates/concordium-std>`_
 
-If you want to familiarize yourself with how proofs work and can be constructed, you can use the `Concordium Proof Explorer <https://web3id-proof-explorer.testnet.concordium.com/>`__ to create proofs and send them to a |bw| to see how they interact with account credentials and verifiable credentials. The Concordium Proof Explorer works on Testnet. You can use the `Web3Id Issuer Front End <https://web3id-issuer-frontend.testnet.concordium.com/>`__ to create verifiable credential to test with the proof explorer.
+.. dropdown:: Proofs
 
-.. _example-dapps:
+    - See :ref:`How to create proofs for dApps and services <create-proofs>` for information about how to write statements that interact with Concordium wallets.
```

Comment:
**Suggestion:** Half of this dropdown seems to be in a list, but the other half is not.
Maybe don't use a list here, since this seems to follow the other dropdowns.